### ISSUE TITLE
Impossible to build

### ISSUE THREAD

**Issue by Dead0utside** [Unipisa/Simu5G#131]
error: unable to load '../out/clang-release/src/libsimu5g.dll.a': No such file or directory
make[1]: *** [Makefile:409: ../out/clang-release/src/libsimu5g.dll] Error 1
make[1]: Leaving directory '.../omnet++/omnetpp-6.0.1/samples/Simu5G-1.2.1/Simu5G-1.2.1/src'
make: *** [Makefile:2: all] Error 2
"make MODE=release all" terminated with exit code 2. Build might be incomplete.

The problem has been reported multiple times but none of the existing solutions help. I'm using windows 10 and the latest versions of omnet++, inet and simu5g

**Comment by Jiacheng188** [comment]
Try to use the master branch, this may help.

**Comment by Dead0utside** [comment]
@Jiacheng188 thanks for your answer but I am using it already

**Comment by Jiacheng188** [comment]
But why the name is "Simu5G-1.2.1/Simu5G-1.2.1" rather than "Simu5G-master", have you changed the name? (I am not sure if the name matters)
I have build simu5g both in windows 10 and ubuntu 20.04 with Simu5G-master under the samples directory.

**Comment by Dead0utside** [comment]
@Jiacheng188 ATM I am using the version from simu5g's website (which is the same as master branch apparently). I have tried using master branch too but the Issue was the same. Did you import the simu5g-master through git import in omnet++ or did you download the master branch and import it as an existing project?

**Comment by Jiacheng188** [comment]
I download the master branch and import it as an existing project.

**Comment by dekaio** [comment]
Is Inet included in your Simu5G project references?
Is the src/makemake configured correctly?

**Comment by Dead0utside** [comment]
Thank you everyone for suggestions, the issue is now solved. Turnes out that 7-zip I've used to unpack the archives messed something up while doing so and this was the reason why simu5g didn't build. Same goes to default windows archive manager. Winrar worked just fine in this regard, so if anone else encounters this problem in future - try using a different way to unpack projects. Have a good day
